site/.env.test

```diff
@@ -0,0 +1,7 @@
+SEARCH_URL='https://search-marble-elasticsearch-test-e3urdt7kb667o7verxgn6bjoee.us-east-1.es.amazonaws.com'
+SEARCH_INDEX='test_index'
+USER_CONTENT_PATH='https://uc-test.lib.com/'
+AUTH_CLIENT_URL='https://okta.nd.edu'
+AUTH_CLIENT_ID='0oa1f3ut0aKpdwap5357'
+AUTH_CLIENT_ISSUER='https://okta.nd.edu/oauth2/ausxosq06SDdaFNMB356'
+S3_DEST_BUCKET="ci-bucket"
```

Comment:
Should these be in clear text?

Reply:
This probably needs to change.  I want to do it as a change to the local build process where you can build a local env file from ssm.  

I will do this in a pr after these have been merged.  This has been here for a very long time already.